ft_singleplotER.m

```diff
@@ -273,15 +273,16 @@
 
 % apply baseline correction
 if ~strcmp(cfg.baseline, 'no')
+  tmpcfg = removefields(cfg, {'inputfile', 'reproducescript'});
```

Comment:
it would be better to explicitly keep the fields that the baseline function expects, rather than excluding the ones it should not have. In the current code (also with your change) there are still many cfg options that end up in the baseline function which should not be there